Tests/SentryTests/SentryTests.m

```diff
@@ -30,21 +30,6 @@ - (void)setUp
     [SentrySDK.currentHub bindClient:nil];
 }
 
-- (void)testVersion
-{
-    NSDictionary *info = [[NSBundle bundleForClass:[SentryClient class]] infoDictionary];
-    NSString *version = [NSString stringWithFormat:@"%@", info[@"CFBundleShortVersionString"]];
-    if ([info[@"CFBundleIdentifier"] isEqualToString:@"io.sentry.Sentry"]) {
-        // This test is running on a bundle that is not the SDK
-        // (code was loaded inside an app for example)
-        // in this case, we don't care about asserting our hard coded value matches
-        // since this will be the app version instead of our SDK version.
-        XCTAssert([version isEqualToString:SentryMeta.versionString],
-            @"Version of bundle:%@ not equal to version of SentryMeta:%@", version,
-            SentryMeta.versionString);
-    }
-}
```

Comment:
This test will fail for beta versions, because the `CFBundleShortVersionString` will not have the beta information.
Technically this test Is really not necessary since we dont use `CFBundleShortVersionString` from the SDK bundle for anything, since the SDK version comes from SentryMeta.

Reply:
Thanks for the comment.